packages/core/cms/faststore/sections.json

```diff
@@ -2166,8 +2166,8 @@
         },
         "editLabel": {
           "title": "Location edit label",
-          "type": "string",
-          "default": "Edit"
+          "default": "",
+          "type": "string"
```

Comment:
```suggestion
          "type": "string"
          "default": "",
```

question: s this the case when we don't want to set/display this button?

Reply:
In the cms display as option-1 (empty)
<img width="400" alt="image" src="https://github.com/user-attachments/assets/042d9b78-1336-48cc-aab9-4db0cfa97146" />

Reply:
Yes, it is.
And by sending empty string the button icon should not be displayed.